docs/configuration.md

```diff
@@ -280,6 +289,39 @@ module.exports = {
 };
 ```
 
+### Custom headers for running the wizard
+
+If you're running the `lhci server` behind an reverse proxy or any other component that requires some `extra headers` you can configure them in the wizard section `extraHeaders`.
+
+```json
+{
+  "ci": {
+    "wizzard": {
+      "extraHeaders": "{\"Authorization\": \"Basic content\"}"
+    }
+  }
+}
+```
+
+> **_NOTE:_** The `wizard` options will be overriten by the `upload` options, to be sure that the wizard options will be used, create a seperate config file
+
+
+### Default lhci server location for running the wizard
+
+If you're running the `lhci wizard` multiple times, you can configure a default `serverBaseUrl` that you have to type them in at each `lhci wizard` run.
```

Comment:
```suggestion
If you're running the `lhci wizard` multiple times, you can configure a default `serverBaseUrl` to avoid typing it in at each `lhci wizard` run.
```